api/integrations/common/models.py

```diff
@@ -28,7 +27,6 @@ class Meta:
         abstract = True
 
     @hook(AFTER_SAVE)
-    @hook(AFTER_DELETE)
```

Comment:
Why did we remove this? Is it because `save` gets called anyway?

Reply:
Yeah, exactly. Now that it's soft deleted, `delete` performs a `save`. Otherwise it gets called twice (because it still triggers the `post_delete` hook as well.